python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -163,6 +168,18 @@ class DistributeTranspiler(object):
                                                                 pserver_program)
            elif role == "TRAINER":
                 trainer_program = t.get_trainer_program()
+           
+           # for nccl2 mode
+           config = fluid.DistributeTranspilerConfig()
+           config.mode = "nccl2"
+           t = fluid.DistributeTranspiler(config=config)
+           t.transpile(trainer_id, workers=workers, current_endpoint=curr_ep)
```

Comment:
workers？it should be trainers?

Reply:
Will update in next PR since this PR need API approvals